scripts/create_tables.sql

```diff
@@ -1 +1,29 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  specialty TEXT
+);
+
+CREATE TABLE patients(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  insurance TEXT
+);
+
+CREATE TABLE appointments(
+  id SERIAL PRIMARY KEY,
+  date DATE,
+  time TIME,
+  comment TEXT,
+  patient_id INTEGER REFERENCES patient(id),
```

Comment:
This won't run because `patient(id)` refers to the `patient` table, which doesn't exist.  It should be `patients(id)`.